refined-github/refined-github
# Fix console errors in PR merging box features

Fixes #5547 

## Test URLs

1. Click on the merge button at the bottom of the page
2. Click on "Cancel" and reload the page
3. No console errors should be produced

## Screenshots

See the review comments below.
